### ISSUE TITLE
Impossible to print on Brother printers, 32-bit libraries are missing

### ISSUE THREAD

**Issue by InfoLibre** [linuxmint/linuxmint#291]
With Linux Mint MATE 20 (not 19.3), it's impossible to print with a lot of Brother printer and all printers that are using 32-bit driver.
Could you add the missing packages (see list at the bottom of https://forums.linuxmint.com/viewtopic.php?f=51&t=325847) ?
After adding these packages, printers work again. But I'm not 100% sure all of these packages are necessary and if packages are missing in this list.

**Comment by SimonIremonger** [comment]
I think, 64bit mint-20 system correctly installs without any 32-bit libraries, this is not a fault. You (might) have been using 32bit mint 19.3 before and/or may have already had ia32-libs installed already for some other reason.

I think, fundamentally right solution is for deb's you mention e.g.    hl3170cdwcupswrapper-1.1.4-0.i386.deb   to be updated to have correct dependencies in them so the right packages get automatically installed via package manager...

I don't (think) this is really a mint-issue, though there is possibility of driver-manager sorts potentially offering to install right things for you, or so...

**Comment by InfoLibre** [comment]
I updated a computer from Linux Mint 19.3 to Linux Mint 20 and I can print.
On another computer where I did not keep Linux Mint 19.3 (formatted all partitions) and installed Linux Mint 20, printers don't work until I manually add the missing 32-bit libraries.
Mainly all printers drivers are 32-bit and no 64-bit driver will never available. You can't say it's not Mint fault and let users without printing or they will choose another Linux distribution. You should reconsider your answer and add the missing libraries.

**Comment by InfoLibre** [comment]
I tested with a new installation of Linux Mint MATE 20. Adding the package lib32stdc++6 is enough to fix the problem, as explained on https://support.brother.com/g/b/faqend. ... 100678_000

**Comment by SimonIremonger** [comment]
I don't think this is mint-specific, loads of distros don't install 32-bit-libraries at all, not even offer 32bit system install at all!   Good reasons to ask them to update their drivers!.   I wonder why they have so many printers not listed in openprinting.org, and annoying drivers...
I can see why I won't be selecting a Brother printer for friends' who need another!.

**Comment by InfoLibre** [comment]
Brother printers work very well with Linux. And when you call the support for a problem under Linux, they help you.
